OrleansDashboard/Implementation/GrainInteractionFilter.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.Runtime;
+using OrleansDashboard.Metrics;
+using OrleansDashboard.Model;
+
+namespace OrleansDashboard.Implementation
+{
+    public sealed class GrainInteractionFilter : IIncomingGrainCallFilter, IOutgoingGrainCallFilter
```

Comment:
Can we not extend the current filter for that? Because we have some logic there that we should keep. Like custom method names and attributes to opt out from the profiler.